lib/vaahextendflutter/widgets/atoms/buttons.dart

```diff
@@ -381,9 +383,11 @@ class ButtonIcon extends StatelessWidget {
               ? null
               : Border.all(
                   width: 2.0,
-                  color: color ?? getColorForButtonType(buttonType ?? ButtonType.primary)!,
+                  color: color ??
```

Comment:
revert this; everything looks good.

Reply:
I don't think your ide has s=max line length set to 100

Reply:
reverted & max line length set to 100